src/Compilers/CSharp/Test/Symbol/Compilation/UsedAssembliesTests.cs

```diff
@@ -3629,7 +3633,8 @@ public static void Main()
 
             void verify(MetadataReference reference0, MetadataReference reference1, string source)
             {
-                AssertUsedAssemblyReferences(source, reference0, reference1);
+                var references = new[] { reference0, reference1 };
+                AssertUsedAssemblyReferences(source, references, references, parseOptions: TestOptions.Regular11);
```

Comment:
>TestOptions.Regular11 [](http://example.com/codeflow?start=91&length=21)

I think we should add a test targeting Preview with ```unsafe``` in the right places

Reply:
This is an expensive test (used assemblies tests hard-code LangVer 10 in some helpers) and I don't think the PR affects used assemblies (the change is to suppress some diagnostics or not depending on language version).

Reply:
> This is an expensive test (used assemblies tests hard-code LangVer 10 in some helpers) and I don't think the PR affects used assemblies (the change is to suppress some diagnostics or not depending on language version).

If you are referring to
```
            var parseOptions = ((CSharpParseOptions)tree.Options).WithLanguageVersion(LanguageVersion.CSharp10);
            yield return (comp.ReplaceSyntaxTree(tree, CSharpTestBase.Parse(builder.ToString(), tree.FilePath, parseOptions)), before, after);
``` 

then, I think we can change that code to
```
            LanguageVersion treeLanguageVersion = ((CSharpParseOptions)tree.Options).LanguageVersion;
            var parseOptions = ((CSharpParseOptions)tree.Options).WithLanguageVersion(treeLanguageVersion > LanguageVersion.CSharp10 ? treeLanguageVersion : LanguageVersion.CSharp10);
            yield return (comp.ReplaceSyntaxTree(tree, CSharpTestBase.Parse(builder.ToString(), tree.FilePath, parseOptions)), before, after);
```

 It looks like the intent is just to be able to parse global usings.  

I am not comfortable to leave these scenarios behind without coverage for the latest version.